toolchain/check/testdata/class/fail_addr_self.carbon

```diff
@@ -10,17 +10,17 @@ class Class {
 }
 
 fn F(c: Class, p: Class*) {
-  // CHECK:STDERR: fail_addr_self.carbon:[[@LINE+6]]:6: ERROR: `addr self` method cannot be invoked on a value.
+  // CHECK:STDERR: fail_addr_self.carbon:[[@LINE+6]]:3: ERROR: `addr self` method cannot be invoked on a value.
   // CHECK:STDERR:   c.F();
-  // CHECK:STDERR:      ^
+  // CHECK:STDERR:   ^~~~
```

Comment:
This is highlighting the `(` but not the `)`, which seems a bit surprising. I think any of these would be OK:

```
 c.F();
 ^~~

 c.F();
 ^~~~~

 c.F();
    ^
```

(Same for other calls in this file.)